tasks.py

```diff
@@ -1466,6 +1466,7 @@ def generate_deepcopy():
 
 
 def _align_helm_crds(source, output):
+    run("go install github.com/mikefarah/yq/v4@v4.45.1")
```

Comment:
Hi, I see that there is a pattern in that file how to fetch/use binaries
https://github.com/metallb/metallb/blob/2838788528ea49787533723eb3f77e04bc3238de/tasks.py#L1230 

I think it would look nicer if we do the same for yq. Thanks

Reply:
We use the "go install" pattern for tools built with go.

Reply:
After taking a closer look, I see there's just [one instance](https://github.com/metallb/metallb/blob/5d0b8ba3c08298a75e9bfc4bf87c27f49b506b9c/tasks.py#L1481) of "go install" used without `fetch_dependency`. And that might have been an oversight. :grimacing: 

I'll update the PR to use the `fetch_dependency` pattern. Thanks!

Reply:
Done in https://github.com/metallb/metallb/pull/2680/commits/57e17f544f3f92caf0003a59867c57da4b330b9b!